private/solr-conf/synonyms_und.txt

```diff
@@ -11,35 +11,8 @@
 
 phone,telephone
 tv,television
-911 call,911 calls,dispatch call,dispatch calls,emergency call,emergency calls
-address,addresses,location,locations
 adu,accessory dwelling unit
-appeal,appeals,petition,petitions
-building permit,building permits,construction permit,construction permits
-business license,business registration,business permit
-business search=>business lookup
-certificate of compliance,certificate of conformance,certificate of conformity
-classes,lessons,instructions,training
-directory,catalogue,roster
-employee,staff,personnel
 garbage,rubbish,waste,trash,debris,litter
-gym,gymns,gymnasium,gynmnasiums,health club,health clubs,fitness club,fitness clubs,fitness center,fitness centers
-internship,internsips,apprenticeship,apprenticeships,traineeship,traineeships,work experience
-job,jobs,career,careers,employment
-job opening,job openings,job posting,job postings,job listing,job listings
-non emergency,non urgent,elective
-parking,car park
-pay bill,pay invoice,pay account,pay notice,pay charge,pay statement
-pay ticket,pay fine,settle ticket
-permit,permits,license,licenses
```

Comment:
I'd recommend keeping this one too as community members probably don't know what's a license or permit.

Reply:
This is true, but "permit" can be used as a verb as well as the noun "permit" that is only a loose synonym for license. In our case, the only thing we have called a "license" in the city is associated with a business license, while permit is associated with a ton of other things. I think we are safer not using it at the moment and boosting the things we find could use the other word by including that word in the summary and/or search keywords.